packages/synapse-react-client/src/components/widgets/facet-nav/SelectionCriteriaPills.tsx

```diff
@@ -135,6 +139,20 @@ function getPillPropsFromQueryFilters(
   })
 }
 
+function getRangeFacetInnerText(min?: string, max?: string) {
+  if (min == undefined && max == undefined) {
+    return 'Any value'
+  } else if (min == undefined) {
+    return `Up to ${max}`
+  } else if (max == undefined) {
+    return `${min} or greater`
+  } else if (min === VALUE_NOT_SET && max === VALUE_NOT_SET) {
+    return FRIENDLY_VALUE_NOT_SET
+  } else {
+    return `${min} - ${max}`
+  }
+}
+
```

Comment:
Handle this bug 

![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/299149f4-dc9d-4e0c-872a-09d5a811b855)

Reply:
<img width="429" alt="image" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/9ac11f22-1058-4c63-b1a0-395feb69dbe5">